units/cache_historical_test_data.go

```diff
@@ -33,6 +33,7 @@ type cacheHistoricalTestDataJob struct {
 	ProjectID       string   `bson:"project_id" json:"project_id" yaml:"project_id"`
 	Requesters      []string `bson:"requesters" json:"requesters" yaml:"requesters"`
 	DisableOldTasks bool     `bson:"disable_old_tasks" json:"disable_old_tasks" yaml:"disable_old_tasks"`
+	UseMerge        bool     `bson:"use_merge,omitempty" json:"use_merge,omitempty" yaml:"use_merge,omitempty"`
```

Comment:
I may be missing a service flag for configuring use merge. It defaults to off for now.